test/commands/plugins/index.test.ts

```diff
@@ -26,19 +27,21 @@ describe('command', () => {
   .do(output => expect(output.stdout).to.equal('No plugins installed.\n'))
   .it('installs and uninstalls @oclif/example-plugin-ts with tags')
 
-  test
-  .command(['plugins:install', 'aliasme'], {reset: true})
-  .stdout()
-  .command(['plugins'], {reset: true})
-  .do(output => expect(output.stdout).to.contain('oclif-debug'))
-  .stdout()
-  .command(['debug'], {reset: true})
-  .do(output => expect(output.stdout).to.contain('debug'))
-  .command(['plugins:uninstall', 'oclif-debug'])
-  .stdout()
-  .command(['plugins'], {reset: true})
-  .do(output => expect(output.stdout).to.equal('No plugins installed.\n'))
-  .it('installs via an alias')
+  if (platform() !== 'win32') {
+    test
```

Comment:
these fail on windows with the err "CannotFindNodeExecutable: Cannot locate node executable" due to process.env.PATH being set to an empty string.

`shelljs.which` looks for dirs in `process.env.PATH` but (I think) fancy-test is modifying.
https://github.com/shelljs/shelljs/blob/master/src/which.js#L58

tried passing ENV as process.env.PATH but the tests are executed through yarn which adds a few paths at the start of the array and end up messing the test.
https://github.com/oclif/fancy-test#environment-variables